### PR TITLE
Fixed pep8 whining in module packages

### PR DESCRIPTION
**Pull request by agaida**

### CODE REVIEW

File: src/modules/packages/main.py [hunk 1]
```diff
@@ -146,42 +162,48 @@ def run_operations(pkgman, entry):
...
+            # we make a separate package manager call for each package so a
+            # single failing package won't stop all of them
             for package in entry[key]:
                 if isinstance(package, str):
                     try:
                         pkgman.install([package])
                     except subprocess.CalledProcessError:
-                        libcalamares.utils.debug("WARNING: could not install package {}".format(package))
+                        warn_text = "WARNING: could not install package "
+                        libcalamares.utils.debug(warn_text + package)
```
Comment (shadeslayer) [inline src/modules/packages/main.py]: Perhaps better to do :

warn_text = "foo"
warn_text += package
debug(warn_text)

File: src/modules/packages/main.py [hunk 2]
```diff
@@ -146,42 +162,48 @@ def run_operations(pkgman, entry):
...
                 else:
                     try:
                         pkgman.run(package["pre-script"])
                         pkgman.install([package["package"]])
                         pkgman.run(package["post-script"])
                     except subprocess.CalledProcessError:
-                        libcalamares.utils.debug("WARNING: could not install packages {}".format(package["package"]))
+                        warn_text = "WARNING: could not install packages "
+                        libcalamares.utils.debug(warn_text
```
Comment (shadeslayer) [inline src/modules/packages/main.py]: Likewise here

File: src/modules/packages/main.py [hunk 3]
```diff
@@ -146,42 +162,48 @@ def run_operations(pkgman, entry):
...
         elif key == "remove":
             pkgman.remove(entry[key])
         elif key == "try_remove":
             for package in entry[key]:
                 try:
                     pkgman.remove([package])
                 except subprocess.CalledProcessError:
-                    libcalamares.utils.debug("WARNING: could not remove package {}".format(package))
+                    warn_text = "WARNING: could not remove package "
+                    libcalamares.utils.debug( warn_text + package)
```
Comment (shadeslayer) [inline src/modules/packages/main.py]: Likewise here.

**Comment by agaida** [comment]
full ack

**Comment by shadeslayer** [comment]
lgtm

**Comment by philmmanjaro** [comment]
@shadeslayer: who is approving in the end when @teo is not always around? Is there a new process?

**Comment by shadeslayer** [comment]
@philmmanjaro I usually merge if I don't hear back from @teo in a week. There isn't a formal process yet, I can let you know what's' going to happen in a couple of weeks though :)

**Comment by philmmanjaro** [comment]
@shadeslayer: sure, @teo told me already some about it. I just was wondering about it in general ...

**Pull request merged**